kong/api/routes/health.lua

```diff
@@ -0,0 +1,88 @@
+local utils = require "kong.tools.utils"
+
+local find = string.find
+local select = select
+local tonumber = tonumber
+local kong = kong
+local knode  = (kong and kong.node) and kong.node or
+               require "kong.pdk.node".new()
+
+
+local select = select
+local tonumber = tonumber
+local kong = kong
+local shms = {}
+
+for shm_name, shm in pairs(ngx.shared) do
+  table.insert(shms, {
+    zone = shm,
+    name = shm_name,
+    capacity = shm:capacity(),
+  })
+end
```

Comment:
The above loop and `shms` variable are unused and seem to be copied from the PDK's `node.lua` module? That doesn't seem to be part of the refactor commit (or rather, it shouldn't), given I don't currently see it in today's Admin API code.